liquidctl/cli.py

```diff
@@ -8,6 +8,7 @@
   liquidctl [options] set <channel> speed <percentage>
   liquidctl [options] set <channel> color <mode> [<color>] ...
   liquidctl [options] set <channel> screen <mode> [<value>]
+  liquidctl [options] set <channel> tempoffset [<value>]
```

Comment:
Could we make this generic over the "property name", in order to avoid having to continuously add new commands for very-device-specific features (see also #297)?

Additionally, at this point this CLI consistently favors fully spelled out names and avoiding abbreviations: e.g. `temperature-offset`.  (If this remains a command, `temperature offset` would also be an alternative).

Reply:
Sorry for the delay, I've just rebased this branch to stay on top of things.

I've looked at the linked issue and the related ones, and AFAICS: there isn't a clear way to do that currently, but it should be resolved somehow because we're hitting into it already. I'm not quite sure what you mean by "property name" - do you maybe mean that there should be a `_device_set_misc()` handler that the drivers would use to their will? `speed`, `color` and `screen` are general enough, but now we have temp offsets (that other devices don't) for starters.

The aquacomputer devices can, off the top of my head, have one fan configured to follow another, use PID calculations for setting fan speed, save retrieve/sensor names etc, which I don't think has been the case for the devices that are currently supported (or at least, not implemented yet). I'm listing these features so we have an idea of what can be supported and plan the CLI approach for supporting that (I'd say) relatively unique stuff.

Reply:
> I'm not quite sure what you mean by "property name" - do you maybe mean that there should be a `_device_set_misc()` handler that the drivers would use to their will?

Yeah, something like that.

By property name I meant modelling these miscellaneous features as name-value pairs:

```py
# private for now
def _set_property(self, name: str, value: Any):
    # ...
```

It should support `value` being either:
- a string, for when it's coming from the CLI, which won't know how to parse it;
- or a suitably typed value for the given driver × `name` pair, for other/direct callers.

And on the CLI side:

```
liquidctl [options] set <property> <value>
```

***

Also maybe

```
liquidctl [options] set <property> <value> ...
```

if we feel that lists are common enough. But then `_set_property` also needs to handle when `value` is a list of strings.